packages/@okta/vuepress-theme-prose/components/ThemeModeToggle.vue

```diff
@@ -24,20 +25,21 @@ export default {
 
   methods: {
     getDarkMode: function() {
-      this.isDarkMode = localStorage[themeModeCookieName] 
-          ? JSON.parse(localStorage[themeModeCookieName])  
+      this.isDarkMode = storage.getItem(THEME_MODE_KEY) 
+          ? JSON.parse(storage.getItem(THEME_MODE_KEY))  
           : window.matchMedia('(prefers-color-scheme: dark)').matches;
```

Comment:
while we're watching for compatible code, `matchMedia` is fairly well supported but not available on IE9: https://caniuse.com/mdn-api_window_matchmedia

Not sure if we care, but we could add safety around this as well